buildutils/vapor.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Name=VAPOR
+Comment=Visualization and Analysis Platform for Ocean, Atmosphere, and Solar Researchers
+Exec=bin/vapor
+Icon=VAPOR
+Categories=Office;
```

Comment:
We probably want to use a more appropriate category like `Science;DataVisualization`